tools/__tasks__/commercial/graph/output/standalone.commercial.ts.json

```diff
@@ -736,13 +736,15 @@
  "../projects/common/modules/identity/auth-component-event-params.js": [
   "../lib/url.ts"
  ],
- "../projects/common/modules/spacefinder-debug-tools.js": [],
+ "../projects/common/modules/spacefinder-debug-tools.ts": [
+  "../projects/common/modules/spacefinder.ts"
+ ],
  "../projects/common/modules/spacefinder.ts": [
   "../../../../node_modules/@guardian/libs/dist/types/index.d.ts",
   "../../../../node_modules/@types/lodash-es/index.d.ts",
   "../lib/fastdom-promise.js",
   "../projects/commercial/am-i-used.ts",
-  "../projects/common/modules/spacefinder-debug-tools.js"
+  "../projects/common/modules/spacefinder-debug-tools.ts"
```

Comment:
Assuming this was a rouge commit?

Reply:
good call -- absolutely no idea what happened here! I'll try and work it out